plugin/pkg/scheduler/algorithm/priorities/selector_spreading.go

```diff
@@ -60,26 +60,26 @@ func NewSelectorSpreadPriority(
 }
 
 // Returns selectors of services, RCs and RSs matching the given pod.
-func getSelectors(pod *v1.Pod, sl algorithm.ServiceLister, cl algorithm.ControllerLister, rsl algorithm.ReplicaSetLister, ssl algorithm.StatefulSetLister) []labels.Selector {
-	selectors := make([]labels.Selector, 0, 3)
-	if services, err := sl.GetPodServices(pod); err == nil {
+func (s *SelectorSpread) getSelectors(pod *v1.Pod) []labels.Selector {
+	var selectors []labels.Selector
```

Comment:
i agree with this change -- i think its ok to add this change into the original getSelectors function once you put the logic back (into a function).   i dont see why the `0,3` was declared, but i could be missing something.